xcube/webapi/ows/stac/controllers.py

```diff
@@ -482,47 +495,76 @@ def _get_single_dataset_collection(
             {
                 'rel': 'items',
                 'href': f'{base_url}{PATH_PREFIX}/collections/'
-                        f'{dataset_id}/items',
+                f'{dataset_id}/items',
                 'title': 'feature collection of data cube items'
             },
             {
                 'rel': 'item',
                 'href': f'{base_url}{PATH_PREFIX}/collections/'
-                        f'{dataset_id}/items/{DEFAULT_FEATURE_ID}',
+                f'{dataset_id}/items/{DEFAULT_FEATURE_ID}',
                 'type': 'application/geo+json',
                 'title': f'Feature for the dataset "{dataset_id}"'
             },
             {
                 'rel': 'http://www.opengis.net/def/rel/ogc/1.0/coverage',
                 'href': f'{base_url}{PATH_PREFIX}/collections/'
-                        f'{dataset_id}/coverage',
+                f'{dataset_id}/coverage?f=json',
+                'type': 'application/json',
                 'title': f'Coverage for the dataset "{dataset_id}" using '
-                         f'OGC API – Coverages'
-            }
+                f'OGC API – Coverages, as JSON',
+            },
+            {
+                'rel': 'http://www.opengis.net/def/rel/ogc/1.0/coverage',
+                'href': f'{base_url}{PATH_PREFIX}/collections/'
+                f'{dataset_id}/coverage?f=netcdf',
+                'type': 'application/x-netcdf',
+                'title': f'Coverage for the dataset "{dataset_id}" using '
+                f'OGC API – Coverages, as NetCDF',
+            },
+            {
+                'rel': 'http://www.opengis.net/def/rel/ogc/1.0/coverage',
+                'href': f'{base_url}{PATH_PREFIX}/collections/'
+                f'{dataset_id}/coverage?f=geotiff',
+                'type': 'image/tiff; application=geotiff',
+                'title': f'Coverage for the dataset "{dataset_id}" using '
+                f'OGC API – Coverages, as GeoTIFF',
+            },
         ],
         'providers': [],
         'stac_version': STAC_VERSION,
         'summaries': {},
         'title': dataset_id,
         'type': 'Collection',
+        'storageCRS': storage_crs,
+        'crs': available_crss
     }
-    result.update(_get_cube_properties(ctx, dataset_id))
+    result.update(_get_cube_properties(ds_ctx, dataset_id))
     return result
 
 
+def crs_to_uri_or_wkt(crs: pyproj.CRS) -> str:
```

Comment:
I'm not a huge fan of functions where you don't know what you'll get. I acknowledge for its purpose it's fine but still ...

Reply:
I agree it's not pretty, but I can't think of a better way to do it: we want to use a standard URI if a CRS is representable as such, and fall back to a WKT representation if not. The caller at least knows that they'll get a valid string representation of the supplied CRS, and under the circumstances I think that's good enough.